src/styles/index.css

```diff
@@ -0,0 +1,61 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: grey;
+}
+
+.product-card {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+}
+
+.product-card__image {
+  background: center url(../images/imac.jpeg);
```

Comment:
The URL in the background property should be enclosed in quotes according to the code style rules.